docs/advanced-topics/tools-vs-workflows.rst

```diff
@@ -0,0 +1,117 @@
+| Nextflow               | N/A                                          | Any valid Nextflow workflow                         |
++------------------------+----------------------------------------------+-----------------------------------------------------+
+| Galaxy                 | N/A*                                         | Any valid Galaxy workflow                           |
++------------------------+----------------------------------------------+-----------------------------------------------------+
+
+\* There are tools that make up Galaxy workflows from the Galaxy toolbox or ToolShed.
+Dockstore currently does not support registration of these tools.
+
+
++------------------------+------------------------------------------+-------------------------------------------------+
+| Dockstore Support      | Tool                                     | Workflow                                        |
++========================+==========================================+=================================================+
+| "Launch with" support  |  Only via Dockstore CLI                  |  Dockstore CLI & "launch with" buttons on entry |
++------------------------+------------------------------------------+-------------------------------------------------+
+| Versioning             |  Based off of image's tags               |  Based off of branches/tags from Git repository,|
+|                        |                                          |                                                 |
+|                        |                                          |  even if one of the tasks executes in a Docker  |
+|                        |                                          |  image.                                         |
++------------------------+------------------------------------------+-------------------------------------------------+
+
+What about workflows that run in Docker containers?
+---------------------------------------------------
+
+Workflow descriptor files do not require a Docker container. That being said, you can still specify external Docker images
+within the descriptor files, and doing so is generally recommended. Each task (WDL) or step (CWL) can specify a Docker image, which might be totally different from the Docker image used by a previous task/step in the workflow. Because of this, workflows registered on Dockstore that reference Docker image(s) will still follow versioning from GitHub, instead of versioning based on the tags of the Docker image(s).
+
+.. hint::
+    The Cromwell execution engine requires that WDL task specify a Docker image when running on GCP or AWS backends, including Terra. Therefore, if your WDL workflow needs to run on Terra, make sure that each task uses the Docker runtime attribute. For more information see the `Cromwell docs <https://cromwell.readthedocs.io/en/stable/RuntimeAttributes/#docker>`_.
```

Comment:
"that WDL task specify..." -> "that a WDL task specify..." or "that WDL tasks specify..."